packages/js/e2e-utils/src/page-utils.js

```diff
@@ -296,3 +297,24 @@ export const selectOrderAction = async ( action ) => {
 		page.waitForNavigation( { waitUntil: 'networkidle0' } ),
 	] );
 }
+
+
+/**
+ * Evaluate and click a button selector then wait for a result selector.
+ * This is a work around for what appears to be intermittent delays in handling confirm dialogs.
+ *
+ * @param {string} buttonSelector Selector of button to click
+ * @param {string} resultSelector Selector to wait for after click
+ * @param {number} timeout Timeout length in milliseconds. Default 5000.
+ * @returns {Promise<void>}
+ */
+export const clickAndWaitForSelector = async ( buttonSelector, resultSelector, timeout = 5000 ) => {
+	await evalAndClick( buttonSelector );
+	await waitForSelector(
+		page,
+		resultSelector,
+		{
+			timeout
```

Comment:
I noticed this dropped the `timeout:` named parameter, so wanted to check if that is needed or not here, or if this needs to be something like:

```suggestion
			timeout: timeout
```

Reply:
The code in the e2e-utils/src folder is ES6 which supports [Shorthand property names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#new_notations_in_ecmascript_2015). The original is the shorthand of your suggestion.

Reply:
I felt I should confirm this so I added a bit of test code

```
 export const clickAndWaitForSelector = async ( buttonSelector, resultSelector, timeout = 5000 ) => {
+       console.log( { timeout } );
        await evalAndClick( buttonSelector );
....
 console.log
    { timeout: 5000 }
```

Reply:
Thanks for the followup and taking the time to confirm this!